### PR TITLE
DAOS-7448 Test: Adding EC Cell size for full stripes.

### PR DESCRIPTION
**Pull request by ravalsam**
We have EC cell size support so adding the full stripes EC tests
to run with different cell sizes from 64K to 1M.

Signed-off-by: Samir Raval <samir.raval@intel.com>

### CODE REVIEW

File: get_results.py [hunk 1]
```diff
@@ -501,6 +502,7 @@ def __init__(self, csv_file_path, output_style="full"):
             "xfer_size":    "Xfer Size",
             "block_size":   "Block Size",
             "cont_rf":      "Cont RF",
+            "ec_cell_size": "Cell Size",
```
Comment (daltonbohning) [inline get_results.py:505]: I'll also add this to the database table

File: tests.sh [hunk 2]
```diff
@@ -29,7 +29,7 @@ function set_default(){
 # Set undefined/default test params
 set_default NUMBER_OF_POOLS 1
 set_default no_of_ps $(($DAOS_CLIENTS * $PPC))
-set_default CONT_RF 0
```
Comment (daltonbohning) [inline tests.sh:32]: Can we please also keep `set_default CONT_RF 0`? That way we can put the default value of 0 in the database without the database itself having to set a default

File: run_testlist.py [hunk 3]
```diff
@@ -952,19 +972,27 @@ def _expand_variant(self, env, ppc, variant):
...
+        env['EC_CELL_SIZE'] = str(variant)
+
     def run(self):
         for test in self._testlist:
             if not test['enabled']:
                 continue
 
+            #Set default EC cell size variants
+            if 'ec_cellsize_variants' not in test:
+                test['ec_cellsize_variants'] = ['default']
```
Comment (daltonbohning) [inline run_testlist.py:985]: Can cellsize only be specified as a number, or does something like "1KB" work as well?
If it's only a number, would it make sense to use `0` as the default? That way we can store a numerical type in the database.

File: run_testlist.py [hunk 4]
```diff
@@ -952,19 +972,27 @@ def _expand_variant(self, env, ppc, variant):
...
             self._expand_default_env_vars(env, test)
             self._expand_extra_env_vars(env, test)
             ppc = test['ppc']
             for variant in test['testvariants']:
                 self._expand_variant(env, ppc, variant)
-                subprocess.Popen(self._script, env=env)
-
+                for ec_variant in test['ec_cellsize_variants']:
+                    self._ec_cell_size_variant(env, ec_variant)
+                    subprocess.Popen(self._script, env=env)
```
Comment (daltonbohning) [inline run_testlist.py:995]: Making sure I understand this correctly... this runs each testvariant with each ec_cellsize_variant?

**Review by daltonbohning — COMMENTED** [review]
Looks good, but I have some comments pertaining to in-progress database work I'm doing

File: get_results.py (hunk 1 above)
Comment (ravalsam) [reply get_results.py:505]: Sounds good, Thanks. But this work we might want for temporary and later we can use the most common size only.

File: run_testlist.py (hunk 3 above)
Comment (ravalsam) [reply run_testlist.py:985]: That is a good suggestion, The default cell size is 1MB so yes let me change that.

File: run_testlist.py (hunk 4 above)
Comment (ravalsam) [reply run_testlist.py:995]: Correct.

File: tests.sh (hunk 2 above)
Comment (ravalsam) [reply tests.sh:32]: Ah sure

File: get_results.py (hunk 1 above)
Comment (daltonbohning) [reply get_results.py:505]: Sure. It's easy to just drop the column later if we don't need it

**Review by daltonbohning — APPROVED** [review]

**Pull request merged**